tests/lib/websocket/test_websocket.py

```diff
@@ -1,24 +1,30 @@
 import uuid
+from unittest.mock import patch
 
 import pytest
 from fastapi import WebSocket
 from websockets.exceptions import ConnectionClosed
 
 from apollo.lib.schemas.message import ShellIOSchema
 from apollo.lib.websocket import ConnectionManager
+from apollo.lib.websocket.interest_type import WebSocketObserverInterestType
 
 
 @pytest.mark.asyncio
 async def test_connect_agent(mocker, websocket_manager):
-    websocket_mock = mocker.patch('fastapi.WebSocket', autospec=True)
-    mock_agent_id = uuid.uuid4()
-
-    await websocket_manager.connect_agent(mock_agent_id, websocket_mock)
-
-    assert (websocket_manager.open_agent_connections[mock_agent_id] is
-            websocket_mock)
-    websocket_mock.accept.assert_awaited_once()
-
+    with patch('apollo.lib.websocket.app.AppConnectionManager.'
+               + 'send_message_to_connections') as patched_function:
+        websocket_mock = mocker.patch('fastapi.WebSocket', autospec=True)
+        mock_agent_id = uuid.uuid4()
+
+        await websocket_manager.connect_agent(mock_agent_id, websocket_mock)
+
+        assert (websocket_manager.open_agent_connections[mock_agent_id] is
+                websocket_mock)
+        websocket_mock.accept.assert_awaited_once()
+        patched_function.assert_called_with(
+            WebSocketObserverInterestType.AGENT_LISTING
+        )
 
```

Comment:
```suggestion


```

Ignoring that Codacy should ideally catch this, please install a flake8 linter in your editor.

I'm trying to spot now but can't be sure I've seen them all.